camel/toolkits/hybrid_browser_toolkit/ts/websocket-server.js

```
@@ -143,7 +143,10 @@ class WebSocketBrowserServer {
 
       case 'get_snapshot_for_ai':
         if (!this.toolkit) throw new Error('Toolkit not initialized');
-        return await this.toolkit.getSnapshotForAI();
+        // Support includeCoordinates and viewportLimit parameters
+        const includeCoordinates = params.includeCoordinates || false;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)